src/articles/2024-01-21_DDEV-for-js.md

```diff
@@ -0,0 +1,327 @@
+---
+title: DDEV For JavaScript Development
```

Comment:
I know that "Node.js development" doesn't totally describe this, but it's a smaller category than just JS.

```suggestion
title: Node.js Development with DDEV
```